zebra/dplane_fpm_nl.c

```diff
@@ -1512,8 +1514,12 @@ static void fpm_process_queue(struct event *t)
 
 	/* Re-schedule if we ran out of buffer space */
 	if (no_bufs) {
-		event_add_event(fnc->fthread->master, fpm_process_queue, fnc, 0,
-				&fnc->t_dequeue);
+		if (wrote_data)
```

Comment:
Isn't enough to rely on `processed_contexts` instead of this additional `wrote_data` variable?

Reply:
yeah I'll do that